frontend/src/index.pug

```diff
@@ -127,55 +127,61 @@ html
               label granularity
             .mui-textfield.summary-picker__date
               input(v-if="isSafariBrowser", type="text", placeholder="yyyy-mm-dd",
-                  v-bind:value="filterSinceDate", v-on:keyup.enter="updateTmpFilterSinceDate",
-                  onkeydown="formatInputDateOnKeyDown(event)", oninput="appendDashInputDate(event)", maxlength=10)
+                v-bind:value="filterSinceDate", v-on:keyup.enter="updateTmpFilterSinceDate",
+                onkeydown="formatInputDateOnKeyDown(event)", oninput="appendDashInputDate(event)", maxlength=10)
               input(v-else, type="date", v-bind:value="filterSinceDate", v-on:input="updateTmpFilterSinceDate",
-                  onkeydown="return false", v-bind:min="minDate", v-bind:max="filterUntilDate")
+                onkeydown="return false", v-bind:min="minDate", v-bind:max="filterUntilDate")
               label since
             .mui-textfield.summary-picker__date
               input(v-if="isSafariBrowser", type="text", placeholder="yyyy-mm-dd",
-                  v-bind:value="filterUntilDate", v-on:keyup.enter="updateTmpFilterUntilDate",
-                  onkeydown="formatInputDateOnKeyDown(event)", oninput="appendDashInputDate(event)", maxlength=10)
+                v-bind:value="filterUntilDate", v-on:keyup.enter="updateTmpFilterUntilDate",
+                onkeydown="formatInputDateOnKeyDown(event)", oninput="appendDashInputDate(event)", maxlength=10)
               input(v-else, type="date", v-bind:value="filterUntilDate", v-on:input="updateTmpFilterUntilDate",
-                  onkeydown="return false", v-bind:min="filterSinceDate", v-bind:max="maxDate")
+                onkeydown="return false", v-bind:min="filterSinceDate", v-bind:max="maxDate")
               label until
             .mui-textfield.summary-picker__date
               a(v-on:click="resetDateRange") Reset date range
             .summary-picker__checkboxes.summary-picker__section
               label
-                input(type="checkbox", v-model="filterBreakdown", v-on:change="setSummaryHash()").mui-checkbox.summary-picker__checkbox
+                input.mui-checkbox.summary-picker__checkbox(type="checkbox", v-model="filterBreakdown",
+                  v-on:change="setSummaryHash()")
                 span breakdown by file type
-              label.merge-group(v-bind:style="filterGroupSelection === 'groupByNone' ? { opacity:0.5 } : { opacity:1.0 }")
-                input(
-                    type="checkbox",
-                    v-model="isMergeGroup",
-                    v-bind:disabled="filterGroupSelection === 'groupByNone'"
-                ).mui-checkbox.summary-picker__checkbox
+              label.merge-group(v-bind:style="filterGroupSelection === 'groupByNone' ?\
```

Comment:
Break before `v-bind` so no need line wrap.

Reply:
Sorry missed this.